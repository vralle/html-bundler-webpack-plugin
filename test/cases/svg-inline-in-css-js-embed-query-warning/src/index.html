<!DOCTYPE html>
<html>
<head>
  <link href="./styles.css" rel="stylesheet">
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <b>HTML: SVG as output of `generator.dataUrl()` function</b>
  <div><img src="@images/icons/home.svg" class="icon"></div>

  <b>HTML: SVG as svg tag</b>
  <div><img src="@images/icons/home.svg?embed" class="icon"></div>

  <b>CSS: SVG background image as data URL (svgToMiniDataURI)</b>
  <div class="icon-imac"></div>

  <b>CSS: SVG background image as data URL (default base64)</b>
  <div class="icon-imac-base64"></div>

  <b>JS: SVG import as data URL (svgToMiniDataURI)</b>
  <div id="js-svg-data-url"></div>

  <b>JS: SVG import as data URL (default base64)</b>
  <div id="js-svg-data-url-base64"></div>
</body>
</html>