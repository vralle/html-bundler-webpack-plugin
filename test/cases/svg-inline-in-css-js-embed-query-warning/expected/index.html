<!DOCTYPE html>
<html>
<head>
  <link href="styles.5046230a.css" rel="stylesheet">
  <script src="main.a8c1cf86.js" defer="defer"></script>
</head>
<body>
  <b>HTML: SVG as output of `generator.dataUrl()` function</b>
  <div><img src="data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' width='16px' height='16px'%3e%3cpath d='M 8 1.320313 L 0.660156 8.132813 L 1.339844 8.867188 L 2 8.253906 L 2 14 L 7 14 L 7 9 L 9 9 L 9 14 L 14 14 L 14 8.253906 L 14.660156 8.867188 L 15.339844 8.132813 Z M 8 2.679688 L 13 7.328125 L 13 13 L 10 13 L 10 8 L 6 8 L 6 13 L 3 13 L 3 7.328125 Z'/%3e%3c/svg%3e" class="icon"></div>

  <b>HTML: SVG as svg tag</b>
  <div><svg viewBox="0 0 16 16" width="16px" height="16px" class="icon"><path d='M 8 1.320313 L 0.660156 8.132813 L 1.339844 8.867188 L 2 8.253906 L 2 14 L 7 14 L 7 9 L 9 9 L 9 14 L 14 14 L 14 8.253906 L 14.660156 8.867188 L 15.339844 8.132813 Z M 8 2.679688 L 13 7.328125 L 13 13 L 10 13 L 10 8 L 6 8 L 6 13 L 3 13 L 3 7.328125 Z'/></svg></div>

  <b>CSS: SVG background image as data URL (svgToMiniDataURI)</b>
  <div class="icon-imac"></div>

  <b>CSS: SVG background image as data URL (default base64)</b>
  <div class="icon-imac-base64"></div>

  <b>JS: SVG import as data URL (svgToMiniDataURI)</b>
  <div id="js-svg-data-url"></div>

  <b>JS: SVG import as data URL (default base64)</b>
  <div id="js-svg-data-url-base64"></div>
</body>
</html>